server/serializers.py

```diff
@@ -258,17 +505,58 @@ def jsonize_clientside_tab(tab: clientside.TabUpdate) -> Dict[str, Any]:
     return d
 
 
+def _localize_module_message(
+    message_id: str,
+    *,
+    default: Optional[str],
+    ctx: JsonizeModuleContext,
+    arguments={},
+    log_not_found=False,
```

Comment:
Why would we _not_ want to log when a message isn't found?

Reply:
Because not finding the translation of a module spec is normal at this point given the current structure of the rest of the code. The module author may not have opted to use i18n, so the (pybabel) catalog this function accesses (via a `MessageLocalizer`) will not contain the corresponding key.

Reply:
Now that I think of it, `True` would be a better default for this argument

Reply:
There's a difference between "The translation is missing" and "The module isn't translated".

We _always_ want to log if the translation is missing. We _never_ want to log if the module isn't translated. This function has all the information it needs to make this decision (assuming `ModuleLocalizer.for_module_zipfile_and_locale_id()` raises `KeyError`).